openreview/stages/venue_stages.py

```diff
@@ -1411,6 +1424,15 @@ def get_readers(self, conference, number):
             readers.append('${3/signatures}')
 
         return readers
+    
+    def get_nonreaders(self, conference, number):
+        nonreaders = []
+
+        if conference.use_area_chairs and self.Participants.AREA_CHAIRS_ASSIGNED in self.readers:
```

Comment:
so secondary area chairs won't see the ratings? are we sure about this?

Reply:
Well if we want Secondary ACs to be readers of ratings, we would only need to add them to readers in the CustomStage